### PR TITLE
docs: Fix build status CI badge in README

### PR DESCRIPTION
**Pull request by Xuanwo**
### What problem does this PR solve?

image

### What is changed and how it works?

Replaced by new CI service's badge.

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by Renkai — APPROVED** [review]

**Review by gaocegege — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## [Website](https://tikv.org) | [Documentation](https://tikv.org/docs/latest/concepts/overview/) | [Community Chat](https://tikv.org/chat)
 
-[![Build Status](https://internal.pingcap.net/idc-jenkins/buildStatus/icon?job=build_tikv_multi_branch%2Fmaster)](https://internal.pingcap.net/idc-jenkins/job/build_tikv_multi_branch/)
+[![Build Status](https://ci.pingcap.net/buildStatus/icon?job=tikv_ghpr_integration_common_test)](https://ci.pingcap.net/blue/organizations/jenkins/tikv_ghpr_integration_common_test/activity)
```
[the remark this answers is not in the document]
Comment (zhouqiang-cl) [reply README.md:5]: The build seems internal domain now. 
@dianqihanwangzi @purelind what do you think about this?

[the remark this answers is not in the document]
Comment (Xuanwo) [reply README.md:5]: Or we can remove this badge directly?

[the remark this answers is not in the document]
Comment (purelind) [reply README.md:5]: fix this now, try  https://ci.pingcap.net/buildStatus/icon?job=tikv_ghpr_build_master

```shell
[![Build Status](https://ci.pingcap.net/buildStatus/icon?job=tikv_ghpr_build_master)](https://ci.pingcap.net/blue/organizations/jenkins/tikv_ghpr_build_master/activity)
```

**Comment by Xuanwo** [comment]
/cc @BusyJay @zhouqiang-cl 

PTAL

**Comment by Xuanwo** [comment]
BTW, do we just do `cargo build` for this job?

**Review by BusyJay — APPROVED** [review]
Thanks!

**Review by JmPotato — APPROVED** [review]

**Review by sticnarf — APPROVED** [review]
LGTM. Thank you very much!

**Pull request merged**